pypodcastparser/Item.py

```diff
@@ -7,6 +7,9 @@
 import pytz
 import logging
```

Comment:
can you hit these exceptions in the unit tests for coverage

Reply:
I've added a few, a lot of them are hard to throw exceptions for without mocking which pypodcastparser is woefully unequipped for.